www/index.html

```diff
@@ -18,8 +18,10 @@
     <script src="app/app.js"></script>
     <script src="app/services/tables.services.js"></script>
     <script src="app/main/main.controller.js"></script>
-    <script src="app/tables/tables.controller.js"></script>
+    <script src="app/search/search.controller.js"></script>
     <script src="app/outings/outings.controller.js"></script>
+    <script src="app/tables/create.controller.js"></script>
```

Comment:
Per our discussion, we'll change the folder from 'tables' to 'create' for both create.controller.js and create.html.